bindings/rust/s2n-tls/src/lib.rs

```diff
@@ -1,8 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#![allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
Is this an intentional removal?

Reply:
yep. the lint [was moved to nursery](clippy::derive_partial_eq_without_eq).